recipes/kosh/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "kosh" %}
+{% set version = "1.1" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/LLNL/kosh/archive/v{{ version }}.tar.gz
+    sha256: 96a82d9ed402da865741a85fe90eb021bd911864d3d85fe3e3ab273a443868fb
+
+build:
+  skip: True  # [win]
+  number: 0
+  # noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - kosh = kosh.cli.driver:main
```

It looks like `kosh` isn't an entrypoint? https://github.com/LLNL/kosh/blob/3c4b3f9c332f502721439fd75068346d210f27e2/setup.py#L32
```suggestion
  # Can't be noarch due to the use of "scripts" in setup.py
  skip: True  # [win]
  number: 0
  script: {{ PYTHON }} -m pip install . -vv
```